src/main/java/edu/cornell/kfs/sys/xmladapters/AwsStringToJavaDateTimeAdapter.java

```diff
@@ -0,0 +1,29 @@
+package edu.cornell.kfs.sys.xmladapters;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+import java.util.Date;
+
+import org.apache.commons.lang.StringUtils;
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+public class AwsStringToJavaDateTimeAdapter extends XmlAdapter<String, Date> {
+
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss");

```

Comment:
Can you move "yyyy-MM-dd HH:mm:ss" to a constant in CUKFSConstants .... there are some other date format strings in there.